src/install/extract_tarball.zig

```diff
@@ -157,13 +157,14 @@ fn extract(this: *const ExtractTarball, tgz_bytes: []const u8) !Install.ExtractD
     var tmpdir = this.temp_dir;
     var tmpname_buf: [256]u8 = undefined;
     const name = this.name.slice();
-
-    var basename = this.name.slice();
-    if (basename[0] == '@') {
-        if (std.mem.indexOfScalar(u8, basename, '/')) |i| {
-            basename = basename[i + 1 ..];
+    const basename = brk: {
+        if (name[0] == '@') {
+            if (std.mem.indexOfScalar(u8, name, '/')) |i| {
```

Comment:
nit: `strings.indexOfChar(name, '/')` is faster (though it shouldn't matter here)

Reply:
Replaced all usages of `std.mem.indexOfScalar()` and `std.mem.lastIndexOfScalar()` within `src/install`